test/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks.csproj

```diff
@@ -34,10 +36,14 @@
     <None Include="..\data\wikipedia-detox-250-line-data.tsv" Link="Input\wikipedia-detox-250-line-data.tsv">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Condition="Exists('..\data\external\WikiDetoxAnnotated160kRows.tsv')"
-                  Include="..\data\external\WikiDetoxAnnotated160kRows.tsv"
-                  Link="external\WikiDetoxAnnotated160kRows.tsv">
+
+    <TlcResourceFile Update="@(TlcResourceFile)">
+      <Link>external\%(Identity)</Link>
+    </TlcResourceFile>
+
+    <None Condition="Exists('@(TlcResourceFile->'..\data\external\%(Identity)')')"
+                  Include="@(TlcResourceFile->'..\data\external\%(Identity)')">
```

Comment:
(nit) I find it more readable if you line up the attributes vertically. Like this

[https://github.com/dotnet/machinelearning/blob/2c0bb2199c8960dabc130b6f19980b2493a922f4/Directory.Build.targets#L24-L30](https://github.com/dotnet/machinelearning/blob/2c0bb2199c8960dabc130b6f19980b2493a922f4/Directory.Build.targets#L24-L30)